app/lib/clients/vault/certificate.rb

```diff
@@ -1,11 +1,16 @@
 module Clients
   class Vault
     module Certificate
+      extend Clients::Vault::Entity
```

Comment:
how does "extend" help here?

Reply:
I moved the entity-related function there (shared by Certificate and KeyValue now)

Reply:
Sorry I wasn't clear, how is "extend" better than "require"?

Reply:
good question -- I think modules are a little weird because they can't be instantiated directly, so for one module to depend on another, it has to "bring it in" and define those methods, etc, where the parent is being included. the 'extend' ensures that the methods are defined wherever this module is included.

Reply:
got it, thanks